src/main/java/dev/nolij/zson/Zson.java

```diff
@@ -956,7 +1038,7 @@ public void write(@NotNull Map<String, ZsonValue> data, @NotNull Appendable outp
 
 	/**
 	 * Checks if the given string is a valid identifier.
-	 * @param key The string to check.
+	 * @param key The string to check.sadasdf
```

Comment:
I have to assume this was left in unintentionally

Reply:
yes 💀

Reply:
I needed changes in the javadoc to test whether or not doclint was turned off, so I was just adding and removing random stuff